air-quality-ui/system_tests/pages/city_page.ts

```diff
@@ -45,6 +45,10 @@ export class CityPage extends BasePage {
     await siteDeselect.click()
   }
 
+  async waitForIdleCharts(chartElement: Locator) {
```

Comment:
I don't think this needs to be its own function as it just calls the other function